src/Keyboard.css

```diff
@@ -51,7 +51,8 @@
 	flex-basis: 56px;
 }
 
-.keyboard-space {
+.keyboard-space,
+.keyboard-spaceButton{
```

Comment:
Let's  use BEM-methodology for naming classes. We have `.keyboard-button` class, and we can add `.keyboard-button--space` class, `.keyboard-button--shift` and so on